packages/geolocation/src/drivers/mongodb/connection.js

```diff
@@ -0,0 +1,34 @@
+const { MongoClient } = require('mongodb');
+
+const config = require('../../../config/mongodb');
+const { dbOptions } = require('../../utils/constants');
+const Logger = require('../../utils/logger');
```

Comment:
Remeber to change the way we import logger

a) if you need to use Logger const { Logger } = require('@booking-services/shared');
b) if you need to reference logger file @booking-services/shared/src/utils/logger